### DISCUSSION TITLE
TLS Handshake failed after restoring cluster from backup 10.0.2

### DISCUSSION THREAD

**Discussion by gordonswing** [gravitational/teleport#14938] [Q&A]
**Teleport version 10.0.2**

**Steps to reproduce:**
- Create a new cluster with etcd store with new certificate chains and with the same IP addresses
- copy backup.yaml
- run restore
_sudo tctl create -f backup.yaml
role 'access' has been created
role 'auditor' has been created
role 'editor' has been created
user "gordonswing" has been created
certificate authority 'tele-test.example.com' has been updated
certificate authority 'tele-test.example.com' has been updated
certificate authority 'tele-test.example.com' has been updated
certificate authority 'tele-test.example.com' has been updated_

In the journal I see errors, the same error while login via UI.
`connection error: desc = "transport: authentication handshake failed: x509: certificate signed by unknown authority"`

Domains replaced for example.com:
_Jul 27 14:33:05 teleport-test-1 teleport[2723]: 2022-07-27T14:33:05Z INFO [AUTH]      "gordonswing" user created auth/grpcserver.go:976
Jul 27 14:34:30 teleport-test-1 teleport[2723]: 2022-07-27T14:34:30Z WARN [MXTLS:1]   Handshake failed. error:[tls: failed to verify client certificate: x509: certificate signed by unknown authority] multiplexer/tls.go:144
Jul 27 14:34:30 teleport-test-1 teleport[2723]: 2022-07-27T14:34:30Z WARN [PROXY:AGE] Failed to sync reverse tunnels: Get "https://teleport.cluster.local/v2/reversetunnels": remote error: tls: bad certificate. reversetunnel/rc_manager.go:162
Jul 27 14:34:31 teleport-test-1 teleport[2723]: 2022-07-27T14:34:31Z INFO [AUDIT]     user.login addr.remote:62.4.40.45:59166 cluster_name:tele-test.example.com code:T1000I ei:0 event:user.login method:local mfa_device:map[mfa_device_name:otp-device mfa_device_type:TOTP mfa_device_uuid:11dd2aab-27b4-4786-a41f-02d6bbf55f37] success:true time:2022-07-27T14:34:31.332Z uid:7824c3fb-dadb-4010-8f68-1fe2279f6925 user:gordonswing user_agent:Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/103.0.0.0 Safari/537.36 events/emitter.go:263
Jul 27 14:34:32 teleport-test-1 teleport[2723]: 2022-07-27T14:34:32Z INFO [CA]        Generating TLS certificate {0x878aa18 0xc000552f10 1.3.9999.1.7=#131c74656c652d746573742e696e742e636861696e737461636b2e636f6d,CN=gordonswing,O=access+O=editor+O=auditor,POSTALCODE={\"aws_role_arns\":null\,\"db_names\":null\,\"db_users\":null\,\"kubernetes_groups\":null\,\"kubernetes_users\":null\,\"logins\":null\,\"windows_logins\":null},STREET=tele-test.example.com,L=-teleport-nologin-d035c683-3a9c-4bf0-814d-0f43dbb28bd6+L=-teleport-internal-join 2022-07-28 02:34:32.192215253 +0000 UTC [] [] 5 []}. common_name:gordonswing dns_names:[] locality:[-teleport-nologin-d035c683-3a9c-4bf0-814d-0f43dbb28bd6 -teleport-internal-join] not_after:2022-07-28 02:34:32.192215253 +0000 UTC org:[access editor auditor] org_unit:[] tlsca/ca.go:873
Jul 27 14:34:32 teleport-test-1 teleport[2723]: 2022-07-27T14:34:32Z INFO [AUDIT]     cert.create cert_type:user cluster_name:tele-test.example.com code:TC000I ei:0 event:cert.create identity:map[expires:2022-07-28T02:34:32.192215253Z logins:[-teleport-nologin-d035c683-3a9c-4bf0-814d-0f43dbb28bd6 -teleport-internal-join] roles:[access editor auditor] route_to_cluster:tele-test.example.com teleport_cluster:tele-test.example.com traits:map[aws_role_arns:<nil> db_names:<nil> db_users:<nil> kubernetes_groups:<nil> kubernetes_users:<nil> logins:<nil> windows_logins:<nil>] user:gordonswing] time:2022-07-27T14:34:32.196Z uid:16239f9f-3105-48af-98a2-95821a4ebae6 events/emitter.go:263
Jul 27 14:34:33 teleport-test-1 teleport[2723]: 2022-07-27T14:34:33Z WARN [MXTLS:1]   Handshake failed. error:[remote error: tls: bad certificate] multiplexer/tls.go:144
Jul 27 14:34:36 teleport-test-1 teleport[2723]: 2022-07-27T14:34:36Z WARN [MXTLS:1]   Handshake failed. error:[remote error: tls: bad certificate] multiplexer/tls.go:144
Jul 27 14:35:00 teleport-test-1 teleport[2723]: 2022-07-27T14:35:00Z WARN [MXTLS:1]   Handshake failed. error:[tls: failed to verify client certificate: x509: certificate signed by unknown authority] multiplexer/tls.go:144
Jul 27 14:35:00 teleport-test-1 teleport[2723]: 2022-07-27T14:35:00Z WARN [PROXY:AGE] Failed to sync reverse tunnels: Get "https://teleport.cluster.local/v2/reversetunnels": remote error: tls: bad certificate. reversetunnel/rc_manager.go:162_

What did I do wrong?

**Comment by gordonswing** [comment]
Did the same with certs from first cluster from /var/lib/teleport, same unexpected result

**Comment by webvictim** [comment]
How are you restoring here? `teleport start --bootstrap backup.yaml`?
